Equipment/EquipmentDb/Weapons/Ranged.xml

```diff
@@ -57,6 +57,15 @@
 		</source>
 	</melee_weapon>
 
+	<melee_weapon id="2100" phase="1">
+		<info name="Precisely Calibrated Boomstick" type="GUN" slot="RANGED" unique="no" req_lvl="43" item_lvl="48" quality="EPIC" boe="yes" icon="Inv_weapon_rifle_06.png"/>
+		<dmg_range min="38" max="45" speed="1.50"/>
+		<stat type="AGILITY" value="14"/>
```

Comment:
You'll need to bake this into a <stats> element, like this:

```
<stats>
    <stat type="AGILITY" value="14"/>
</stats>
```

This gun currently does not provide any agility otherwise.

Reply:
woops fixed